### DISCUSSION TITLE
Need to restart project for new routes to take effect

### DISCUSSION THREAD

**Discussion by cwandev** [posva/unplugin-vue-router#425] [Questions and Help] [answered]
Hi @posva, I'm currently using your plugin for a Vue 3 project and finding it very convenient! However, I've encountered a small issue. It seems that every time I create a new page, I need to restart the project for the changes to take effect. Am I missing something? I would greatly appreciate your help with this matter. Looking forward to your reply.

### Mini Repo

https://github.com/easy-temps/vue3-vant-mobile

**Comment by posva** [comment]
Hello! I don't see anything in particular regarding the usage of the plugin.
Normally, creating a new page doesn't require a restart but I don't know what could be causing it. If I were you, I would strip down the project or start from a simpler template that also has unplugin-vue-router like my posva/uvr-demo repo

**Reply by cwandev** [reply]
Thank you for your response. I appreciate the suggestion. I'll try stripping down the project or starting from a simpler template, as you mentioned. I'll take a look at the posva/uvr-demo repo and see if that helps resolve the issue. Thanks again for your guidance!

Best regards.

**Comment by cwandev** [comment, marked as answer]
Upgrading to the latest version worked well.

- `vue-router`: "^4.4.0"
- `unplugin-vue-router`: "^0.10.0"
